style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  font-family: Roboto;
+}
+
+.calculator{
+  width: 248px;
+  height: 380px;
+  line-height: 1.5;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  margin: auto;
+  background-color: #adadad;
+  align-items: center;
+  border-radius: 10px;
+  box-shadow: 10px 10px 10px 5px grey;
+
+}
+
+
+.calculator #screen {
```

Comment:
I never use ids. I also never ever ever use them in CSS. When you use an ID you break your ability to have more than on instance of a component on the page. This trade-off is in return for nothing of value. Change this to a unqiue classname like `.calculator-screen` ?